Automattic/wp-calypso
# BlazePress: Set up translation function

Setting up translation function for the DSP Widget.

### Testing
1. Apply this diff.
2. Make sure you have a widget version that includes the patch mentioned in this P2: pMyNb-5Gd-p2 
3. Open the widget in Calypso with an account in a different language than English.
4. Verify you see the main title translated (this works because the `Promote your posts` string is already translated in Calypso):
<img width="636" alt="image" src="https://user-images.githubusercontent.com/375980/187706208-9598bb6a-40f8-4b58-8fc5-7f290e52a9c1.png">
